MainModule/Client/Client.lua

```diff
@@ -257,7 +257,7 @@ GetEnv = function(env, repl)
 			return (locals[ind] or (env or origEnv)[ind])
 		end,
 
-		__metatable = unique,
+		__metatable = Folder:FindFirstChild("ADONIS_DEBUGMODE_ENABLED") and nil or unique,
```

Comment:
This statement will always resolve as `unique`
![image](https://github.com/Epix-Incorporated/Adonis/assets/112895605/2cfeb031-8361-40c0-a918-2d13ff3b056b)

try doing something like `__metatable = if DebugEnabled then nil else unique`

Reply:
please kiss yourself

Reply:
your code doesnt work poo head

Reply:
its not mine :trollface:

Reply:
*it's 😎😎😎😎😎

Reply:
These and the others are fixed in https://github.com/Epix-Incorporated/Adonis/pull/1216/commits/327f85050dac716ec86e09045f455dcb2945988f